controller_coverage.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Usage: bash ./controller_coverage.sh
+# Then run: firefox controller_coverage/index.html
+
+set -e
+set -x
+
+# delete coverage files
+#find .pio -name "*.gcno" -delete
```

Comment:
Delete commented line?  Or do you want to keep this but uncommented?  If so presumably you want to delete only under .pio/build/$ENVIRONMENT?